### DISCUSSION TITLE
Feature Request: SysVar-Scan (ein/aus) wieder in den generellen Config-Flow zurückbringen...

### DISCUSSION THREAD

**Discussion by Baxxy13** [SukramJ/aiohomematic#1716] [Q&A] [answered]
Hi Sukram...

Ich hatte mal wieder ein HA-Testsystem "zerlegt" und musste alles neu machen.
Habe mir gleich die 1.66.0b7 installiert und dann die erste RM integriert.
Dabei ist mir aufgefallen das man im "ersten Durchlauf" durch den Config-Flow die erweiterte Konfiguration nicht aktivieren kann.
Eigentlich nicht so wild, kann man ja im "zweiten Durchlauf" dann machen.

Der Haken ist nun aber, in den erweiterten Optionen steckt das ein/aus für den SysVar-Scan. Die Option ist per default aktiv und ich kann sie im "ersten Durchlauf" nicht deaktivieren. Demzufolge werden alle SysVars eingelesen und als deaktivierte Entitäten angelegt.

Die ganzen SysVar-Entitäten wieder loszuwerden ist mühselig.
- Config-Flow erneut durchlaufen und in den erweiterten Optionen den SysVar-Scan deaktivieren
- alle SysVar-Entitäten "aktivieren", HA neu starten
- alle nun nicht mehr bereitgestellten SysVar-Entitäten löschen

"Früher" (mit 1.64.0 z.B.) fand ich das besser. 
Im "ersten Durchlauf" durch den Config-Flow gleich den SysVar-Scan deaktiviert und es wurden erst gar keine SysVar-Entitäten angelegt.

Kannst du das bitte "zurückbringen"?

**Comment by SukramJ** [comment]
Ich denk drüber nach , wenn ich mal wieder was am ConfigFlow umbauen muss.

**Reply by Baxxy13** [reply]
Alles klar. 
Mir schwirrt da noch was durch den Kopf worüber man ggf. auch mal nachdenken könnte. 😉 

Mache ich eine separate Diskussion zu auf.

**Comment by SukramJ** [comment, marked as answer]
Die erweiterte Konfiguration kann jetzt auch im ConfigFlow genutzt werden, mit der Einschränkung das un_ignore ausgeblendet wird, da noch keine Daten vorhanden sind.

Hab ich mit der [1.66.0b8](https://github.com/danielperna84/custom_homematic/releases/tag/1.66.0b8) umgesetzt.

Bitte testen.

**Reply by Baxxy13** [reply]
Sehr schön, danke!
Hab gleich mal eins der integrierten Testsysteme in HA gelöscht und neu integriert.
Lief problemlos und sauber durch, hatte den SysVar-Scan und den Programm-Scan direkt deaktiviert.
